tox.ini

```diff
@@ -19,6 +19,12 @@ commands =
     sphinx-build -W -b html -d {envtmpdir}/doctrees docs docs/_build/html
     doc8 --allow-long-titles README.rst docs/ --ignore-path docs/_build/
 
+[testenv:manifest]
```

Comment:
Does this need a corresponding `.travis.yml` change?

Reply:
Ha! Yes.